FalmouthFalcons/iteration-3
# Adam file structure

Description:
-Add description about what this pull request covers.
.adding file structure for controllers and models.
.not comprehensive. 
.db relationships are incomplete. 
 

Number of Fixes:
-n/a.

Related Ticket(s):
- none

Problem to Solve:
- README models and application generation instructions.

Proposed Changes:
-n/a.

Expected Behavior:
- n/a

Steps to Test Solution
1)git pull origin master 
2)git fetch  
3)git checkout adam-file-structure

Documentation
[Y ] There is new documentation in this pull request that must be reviewed..
[ N] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
